kind-config.yaml

```diff
@@ -6,3 +6,6 @@ nodes:
   - containerPort: 30000
     hostPort: 30000
     protocol: TCP
+- role: worker
+- role: worker
+- role: worker
```

Comment:
Why is this duplicated?

Reply:
to have multiple workerrs on the kind cluster, this is to test how k8s distributes the nats pods